main.go

```diff
@@ -44,6 +44,7 @@ var (
 	skipForeignKeys  bool
 	sessionJSON      string
 	webapi           bool
+	dumpFile         string
```

Comment:
nit; should we call this dumpFilePath?

Reply:
makes sense. Changed.
I have left the flag as is , calling it dump-file as that seemed more intuitive as a command line parameter

Reply:
sounds good.